core/core.go

```diff
@@ -26,12 +25,9 @@ func Preload() {
 	logger.Info("Preloading services in the background.")
 
 	var waitGroup sync.WaitGroup
-	waitGroup.Add(6) // One for each service
+	waitGroup.Add(5) // One for each service
```

Comment:
Might be nicer to use the length of the funcs list instead of manually specifying?